pkg/infra/README.md

```diff
@@ -3,68 +3,68 @@
 根据Clusterfile里的定义去申请IaaS资源，保障IaaS资源与Clusterfile中的定义保持终态一致
 
 ```yaml
-apiVersion: xxx 
+apiVersion: xxx
 kind: Infra
 metadata:
-  name: alicloud-infra-demo
+  name: aws-infra-demo
+  annotations:
+    sealos.io/VpcID: v-xxxxx
+    sealos.io/RouteID: v-xxxxx
+    sealos.io/IgwID: v-xxxxxx
+    sealos.io/EgwID: v-xxxxxx
+    sealos.io/SubnetID: v-xxxxx
+    sealos.io/SecurityGroupID: v-xxxxx
+    sealos.io/EIPID: v-xxxxx
 spec:
   provider: AliyunProvider
   credential:
     accessKey: xxx
     accessSecret: xxx
   cluster:
-    isSeize: true
     regionIds: [cn-hangzhou, cn-shanghai]
-    zoneIds: [cn-hangzhou-a, cn-hangzhou-b] # If the value is empty, a random value will be used.
-    annotations: #set default value,if exist not delete resource
-        sealos.io/VpcID: v-xxxxx
-        sealos.io/VSwitchID: v-xxxxx
-        sealos.io/SecurityGroupID: v-xxxxx
-    accessChannels: # If all access channels are the same, it will be filled.
+    zoneIds: [cn-hangzhou-a, cn-hangzhou-b]
+    accessChannels:
       ssh:
-        passwd: xxx #If the passwd is empty, a random password will be generated.
+        username: ubuntu
+        # ENUM: passfile/pkfile
+        type: pkfile
+        data: $homedir/.ssh/seolos.pk or $homedir/.ssh/seolos.pass
         port: 22
   hosts:
-    - roles: [master, aaa, bbb]
-      count: 3
-      cpu: 4
+    - roles: [master, aaa, bbb] # required
+      count: 3 # Required
+      cpu: 2
       memory: 4
-      arch: amd64 # ENUM: amd64/arm64 (NOTE: the default value is amd64)
-	  ecsType: "ecs.n1.medium" #fixed ecs type
-      os: # 默认 CentOS 7.6
-        name: CentOS # ENUM: CentOS/Ubuntu/Debain and so on.
-        version: 7.6
-        id: xxx # 允许用户指定已有的镜像ID （Optional）
```

Comment:
ID Is it more appropriate to keep it? Because sometimes users directly fix the OS ID.